.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: Conda Package
+on:
+  push:
+    branches:
+    - '*'
+  #pull_request:
+  #  branches:
+  #  - '*'
+jobs:
+  build:
+    name: Build (${{ matrix.python-version }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'windows-latest']
+        python-version: ['3.6', '3.7', '3.8']
+    steps:
+      - name: "Clone pyOCCT"
+        uses: actions/checkout@v2
+
+      - name: "Install miniconda"
+        uses: goanpeca/setup-miniconda@v1
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.python-version }}
+      - name: "Install OpenGL"
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+            sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+            sudo apt update
+            sudo apt -q -y install g++-10 libglu1-mesa-dev libgl1-mesa-dev libxi-dev  libglfw3-dev libgles2-mesa-dev
+            sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 90 --slave /usr/bin/g++ g++ /usr/bin/g++-10 --slave /usr/bin/gcov gcov /usr/bin/gcov-10
+      - name: "Configure conda"
+        run: |
+            conda info
+            conda config --add channels conda-forge
+      - name: "Install conda-build"
+        run: |
+            conda install python=${{ matrix.python-version }} conda-build conda-verify
+      - name: "Build"
```

Comment:
would it be possible to add a step that saves the source files generated? i'm thinking it would be useful to be able to download everything you need to compile locally as that may help with debugging specific binder/OCCT things, and things can be updated. i guess i can just build locally with conda, and dig out the files there for the same purpose. but, might be worth considering.

Reply:
Yes that's a good idea. 

Locally I have just run been running the binder but not adding the `src/occt` files

Reply:
After thinking about this more I don't think this will work how I have it now.

The binder is run in a separate step before conda build starts. It's probably better to do that way anyways since the conda package will then contain the source it used to build (since it includes the recipe).

What are your thoughts on doing it like that?

Reply:
hmm ya you're right. run the first separately, collect the source files for ref, then build. seems like a logical way to break down the steps.